### ISSUE TITLE
[a11y Design Review] Appointments Team: Appointment Details (No AT Testing)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#65318]
## Auditor(s)

- Kate Murphy @ohgreatkate 

## About Your Team
- **Team name:** Appointments Team
-	**OCTO-DE product owner:** Katherine Lawyer
-	**Product manager:** Jeff Roof and Leah De La Costa
-	**Designer:** Peter Russo, Ciera Maddox and Melissa Lefevre 
- **Accessibility specialist:**
- **Content writer:** 
-	**Slack channel:** #appointments-team

## Supporting artifacts from CAIA Intake Ticket #63219 

- **Product outline**: [Appointments Team Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/vaos-product-outline.md)
- User flows:
- **Prototype or mockups:**
    - [Direct Schedule Details Page Mockups](https://www.figma.com/file/RcDljBwThbwVDkNrDH2xiA/Appointment-Details-Redesign?type=design&node-id=2238%3A89569&mode=design&t=bmbXr8cSFmVmsvVZ-1)
    - [Appointment Request Details Page Mockups](https://www.figma.com/file/RcDljBwThbwVDkNrDH2xiA/Appointment-Details-Redesign?type=design&node-id=2238%3A89572&mode=design&t=bmbXr8cSFmVmsvVZ-1)
- **Content source of truth:**
    - [Direct Scheduled Appointment Content Matrix](https://www.figma.com/file/RcDljBwThbwVDkNrDH2xiA/Appointment-Details-Redesign?type=design&node-id=2103%3A16270&mode=design&t=bmbXr8cSFmVmsvVZ-1)- This lists all proposed content for each direct scheduled appointment details page.
    - [Appointment Request Content Matrix](https://www.figma.com/file/RcDljBwThbwVDkNrDH2xiA/Appointment-Details-Redesign?type=design&node-id=2102%3A17112&mode=design&t=bmbXr8cSFmVmsvVZ-1) - This lists all proposed content for each appointment request details page.
- **Any epics/issues that may be helpful:** [Feature #52872](https://app.zenhub.com/workspaces/appointments---productdesign-5fff340c2d80a4000fb6f69c/issues/gh/department-of-veterans-affairs/va.gov-team/52872)
- **Link to testing accounts** (please do not paste them in this ticket): [Staging users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-vaos.md)

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [x] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [x] WCAG 2.1 AA
  - [ ] WCAG 2.1 AAA
  - [ ] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [ ] High Contrast Mode
- [ ] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [ ] Annotations for Tab order, reflow and reading order (if needed)
- [ ] Accessible Name Annotations
  - [ ] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [ ] Text alternative for complex images such as charts graphs maps etc.
- [ ] Language (ease of reading)
- [ ] HTML Tags
  - [ ] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [ ] [Heading order](https://accessibility.18f.gov/headings/)
   - [ ] Annotations (if needed)
  - [ ] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)](https://design.va.gov/components/table)
- [x] Size of clickable areas

### As Time Allows
- [x] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [x] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.
```[tasklist]
### Review Process \[WIP]
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65471
- [x] Create markdown draft from [Design review template](link) to create final deliverable summary
- [x] Reach out to team for any outstanding resources necessary to complete design review, and agree on how to deliver feedback (comments in Figma, markdown file table, call to verbally review)
- [x] conduct design review and document findings
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65474
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65756
```

**Comment by ohgreatkate** [comment]
_Moving previous updates out of the main ticket details and into comments for better tracking._

Intake Notes:
Core request: For accessibility, we need feedback on the usage of buttons versus links. 

Research will not be conducted with AT users at this time, the prototype is not developed for AT testing.

**Comment by ohgreatkate** [comment]
Removed Defect outline; as feedback will be provided in GitHub through a markdown file, within the CAIA folder.

**Comment by ohgreatkate** [comment]
After initial review of design links, this prototype is in Figma, and does not yet have annotations. We will be able to review for contrast, how well it follows the VADS, and cognitive pitfalls within the UX.

**Comment by ohgreatkate** [comment]
[Summary of findings markdown file](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Audits/Appointments%20FE/Design%20review.md)
